code/controllers/subsystem/garbage.dm

```diff
@@ -29,7 +29,7 @@ SUBSYSTEM_DEF(garbage)
 	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
 	init_order = INIT_ORDER_GARBAGE
 
-	var/list/collection_timeout = list(5 MINUTES, 10 SECONDS) // deciseconds to wait before moving something up in the queue to the next level
+	var/list/collection_timeout = list(GARBAGE_QUEUE_FIRST_LEVEL_TIMEOUT, 10 SECONDS) // deciseconds to wait before moving something up in the queue to the next level
```

Comment:
You should define both, and name them like GC_FILTER_QUEUE and GC_DEL_QUEUE or something that describes their purpose rather then just first and second

Reply:
Or maybe something else, just keep it brief